### ISSUE TITLE
Pass W3C Validation

### ISSUE THREAD

**Issue by yutv** [Smile-SA/elasticsuite#1640]
It would be great the elasticsuite passed the W3C validation because some clients require that.

Here are a couple errors caused by the module:

1. Error: Element input is missing required attribute aria-expanded ([source code](https://github.com/Smile-SA/elasticsuite/blob/2.8.x/src/module-elasticsuite-core/view/frontend/templates/search/form.mini.phtml#L44))
2. Warning: The type attribute is unnecessary for JavaScript resources ([1](https://github.com/Smile-SA/elasticsuite/blob/2.8.x/src/module-elasticsuite-tracker/view/frontend/templates/config.phtml#L20), [2](https://github.com/Smile-SA/elasticsuite/blob/2.8.x/src/module-elasticsuite-tracker/view/frontend/templates/variables/page.phtml#L21))

It is not a complete list, there might be more errors.

**Comment by rbayet** [comment]
Actually, this seems like a complete list.
There are no issues related to layered navigation filters, except when having visual swatches, but that's outside of our module.
